assets/js/blocks/products/all-products/edit.js

```diff
@@ -40,14 +61,50 @@ class Editor extends Component {
 		debouncedSpeak: PropTypes.func.isRequired,
 	};
 
+	state = {
+		isEditing: false,
+		innerBlocks: [],
```

Comment:
State allows us to restore to an earlier, unsaved innerblock state. Otherwise any change to innerblocks persists.

Reply:
> State allows us to restore to an earlier, unsaved innerblock state. Otherwise any change to innerblocks persists.

Hmm I see the mechanism for restoring appears to be a reset button correct? Is the Gutenberg undo/redo not sufficient for this here - or does it not for these changes?